config/boards/rock960.conf

```diff
@@ -0,0 +1,9 @@
+# Rockchip RK3399 core 2GB RAM SoC GBE eMMC USB3 USB-C WiFi/BT
+BOARD_NAME="ROCK960"
+BOARDFAMILY="rk3399"
+BOOTCONFIG="rock960-rk3399_defconfig"
+KERNEL_TARGET="legacy,current,edge"
```

Comment:
Since there is only a patch for `current` branch do images with legacy and edge work OOB?